data/transition-sites/decc_corwm.yml

```diff
@@ -1,8 +1,8 @@
 ---
 site: decc_corwm
-whitehall_slug: department-of-energy-climate-change
+whitehall_slug: department-for-business-energy-and-industrial-strategy
+homepage: https://www.gov.uk/government/organisations/department-for-business-energy-and-industrial-strategy
 host: corwm.decc.gov.uk
 tna_timestamp: 20130503173700
-homepage_furl: www.gov.uk/corwm
-homepage: https://www.gov.uk/government/organisations/committee-on-radioactive-waste-management
+homepage_furl: www.gov.uk/beis
 css: department-of-energy-climate-change
```

Comment:
Removed CSS config in https://github.com/alphagov/transition-config/pull/1204/commits/d6ec0d72e9633eecda14119a2ebc8d0e74de23c9. Should we remove it from Bouncer as well?